metisp/pymetis/src/pymetis/recipes/ifu/metis_ifu_rsrf.py

```diff
@@ -213,6 +213,7 @@ class MetisIfuRsrf(MetisRecipe):
     _author = "Janus Brink"
     _email = "janus.brink27@gmail.com"
     _synopsis = "Determine the relative spectral response function."
+    _copyright = "GPL-3.0-or-later"
```

Comment:
This is a global default, so it does not have to be repeated here (and I don't suppose any recipe will have a different one).